diff -urN xf86-video-radeonhd-1.2.0.org/utils/conntest/Makefile.am xf86-video-radeonhd-1.2.0/utils/conntest/Makefile.am
--- xf86-video-radeonhd-1.2.0.org/utils/conntest/Makefile.am	2008-04-10 21:20:56.000000000 +0200
+++ xf86-video-radeonhd-1.2.0/utils/conntest/Makefile.am	2008-04-12 08:54:05.480569775 +0200
@@ -7,7 +7,7 @@
 EXTRA_PROGRAMS = rhd_conntest rhd_dump
 if HAVE_PCI_PCI_H
 if HAVE_ZLIB
-noinst_PROGRAMS = rhd_conntest rhd_dump
+bin_PROGRAMS = rhd_conntest rhd_dump
 endif
 endif
 
@@ -28,4 +28,4 @@
 
 rhd_dump_SOURCES = rhd_dump.c
 nodist_rhd_dump_SOURCES = git_version.h
-rhd_dump_LDADD = @PCIUTILS_LIBS@
\ Brak znaku nowej linii na końcu pliku
+rhd_dump_LDADD = @PCIUTILS_LIBS@
