diff -urN xf86-video-radeonhd.org/utils/conntest/Makefile.am xf86-video-radeonhd/utils/conntest/Makefile.am
--- xf86-video-radeonhd.org/utils/conntest/Makefile.am	2007-11-05 09:54:44.000000000 +0100
+++ xf86-video-radeonhd/utils/conntest/Makefile.am	2007-11-05 10:18:04.498030747 +0100
@@ -4,7 +4,7 @@
 
 EXTRA_DIST = README Imakefile
 
-noinst_PROGRAMS = rhd_conntest
+bin_PROGRAMS = rhd_conntest
 
 # Including config.h requires xorg-config.h, so we need the XORG_CFLAGS here
 AM_CFLAGS   = @XORG_CFLAGS@ @WARN_CFLAGS@
@@ -12,4 +12,4 @@
 
 rhd_conntest_SOURCES = rhd_conntest.c
 nodist_rhd_conntest_SOURCES = git_version.h
-rhd_conntest_LDADD   = -lz -lpci
+rhd_conntest_LDADD   = -lpci -lz
